src/udp_wrap.cc

```diff
@@ -191,21 +205,8 @@ void UDPWrap::DoBind(const FunctionCallbackInfo<Value>& args, int family) {
   if (!args[1]->Uint32Value(ctx).To(&port) ||
       !args[2]->Uint32Value(ctx).To(&flags))
     return;
-  char addr[sizeof(sockaddr_in6)];
-  int err;
-
-  switch (family) {
-  case AF_INET:
-    err = uv_ip4_addr(*address, port, reinterpret_cast<sockaddr_in*>(&addr));
-    break;
-  case AF_INET6:
-    err = uv_ip6_addr(*address, port, reinterpret_cast<sockaddr_in6*>(&addr));
-    break;
-  default:
-    CHECK(0 && "unexpected address family");
-    ABORT();
-  }
-
+  struct sockaddr addr;
```

Comment:
Should these be `sockaddr_storage` to make sure they are large enough?

Reply:
Yeah, I'm sure you are right. For some reason I thought that this would work without and it did look like it was when stepping through it. I'll update shortly and see if this fixes the ci failures. Thanks!